examples/ola-rdm.cpp

```diff
@@ -615,7 +620,8 @@ int main(int argc, char *argv[]) {
   delete opts.uid;
 
   if (opts.list_pids)
-    DisplayPIDsAndExit(dest_uid.ManufacturerId(), controller.PidHelper());
+    DisplayPIDsAndExit(dest_uid.ManufacturerId(), controller.PidHelper(),
+                       opts.set_mode);
```

Comment:
Nit, can you put each argument on a new line please (i.e. move the PidHelper one onto it's own line). Sorry I should have suggested this in the earlier comment.